docs/cloud/dev/spryker-cloud-commerce-os/preparation-for-going-live.md

```diff
@@ -8,14 +9,79 @@ redirect_from:
   - /docs/en/preparation-for-going-live
 ---
 
-This document describes how to prepare to go live with Spryker Cloud Commerce OS.
+This instruction explains how to prepare a Spryker project for going live.
 
-To go live, prepare the production environment by following the checklist:
+## Spryker go-live preparation
 
-* The desired application version is deployed in the production environment. See [Deploying in a production environment](/docs/cloud/dev/spryker-cloud-commerce-os/deploying-in-a-production-environment.html).
-* The domain name is set up. See [Setting up a custom domain name](/docs/cloud/dev/spryker-cloud-commerce-os/setting-up-a-custom-domain-name/setting-up-a-custom-domain-name.html).
-* The SSL certificate is set up. See [Setting up a custom SSL Certificate](/docs/cloud/dev/spryker-cloud-commerce-os/setting-up-a-custom-ssl-certificate.html).
+When you are preparing your project for go-live, consider the following sections and timeframes. 
 
-## Next step
+{% info_block infoBox "Note" %}
 
-[Setting up a custom domain name](/docs/cloud/dev/spryker-cloud-commerce-os/setting-up-a-custom-domain-name/setting-up-a-custom-domain-name.html)
+The suggested timeframes should serve as orientation and might be different for you. Ensure that in the last week leading up to your go-live, none of the following tasks are still open.
+
+{% endinfo_block %}
+
+## Until five weeks before go-live
+
+If you migrate from another shop or project to Spryker (the domain you want to use already points to a shop or project), you need a migration plan and a plan to phase out your old project and phase in your new one. 
+
+Plan the migration concept from your old project to Spryker thoroughly. Check with your SEO experts on what the strategy for your content and search engine results should be. Prepare this concept before you delegate your DNS to Spryker.
+
+If you delegate DNS to Spryker, inform us when you want us to switch the relevant records from one system to the other.
+
+{% info_block infoBox "Note" %}
+
+DNS propagation is not instant. Therefore, do not "switch off" your old project after the switch is done. To migrate, assign time in your project to plan the switch.
+
+{% endinfo_block %}
+
+When you have delegated DNS to us, you will need to inform us when you want us to switch the relevant records from one system to the other. Also note that DNS propagation is not instant, meaning that you should not "switch off" your old project immediately after the switch was done. Assign time in your project to plan this if you need to migrate.
```

Comment:
The repeats what is stated in lines 30-34. Consider deleting this paragraph

Reply:
Duplicate removed.